README.md

```diff
@@ -532,8 +532,8 @@ MIGRATION: Instead of replacing the whole stats functionality you can now custom
   - MIGRATION: Use `setId` instead
 - acorn 5 -> 6
 - Testing
-  - HotTestCases now run for multiple target `async-node` `node` `web` `webworker`
-  - TestCases now also run for filesystem caching with `store: "instant"` and `store: "pack"`
-  - TestCases now also run for deterministic module ids
+  - HotTestCases now run for multiple targets `async-node` `node` `web` `webworker`
+  - TestCases now also runs for filesystem caching with `store: "instant"` and `store: "pack"`
```

Comment:
`runs` seems to be the grammatically correct one. Can you explain the change here?

Reply:
i changed it to `runs`, also needs changing in line 535.

Reply:
done